addons/project/static/src/project_sharing/views/list/list_renderer.js

```diff
@@ -2,14 +2,14 @@
 
 import { evaluateBooleanExpr } from "@web/core/py_js/py";
 import { ListRenderer } from "@web/views/list/list_renderer";
-import { onWillUpdateProps } from "@odoo/owl";
+import { onWillRender } from "@odoo/owl";
 
 export class ProjectSharingListRenderer extends ListRenderer {
     setup() {
         super.setup(...arguments);
         this.setColumns(this.allColumns);
-        onWillUpdateProps((nextProps) => {
-            this.setColumns(nextProps.archInfo.columns);
+        onWillRender(() => {
+            this.setColumns(this.props.archInfo.columns);
         });
```

Comment:
Why we need to call setColumn in the setup + onWillRender with different param ?